scalameta/trees/shared/src/main/scala/scala/meta/internal/Scaladoc.scala

```diff
@@ -97,10 +97,10 @@ object Scaladoc {
   /* List blocks */
 
   /** Represents a list item */
-  final case class ListItem(text: Text, nested: Option[ListBlock] = None)
+  final case class ListItem(term: Seq[Term])
 
   /** Represents a list block */
-  final case class ListBlock(prefix: String, item: Seq[ListItem]) extends Term
+  final case class ListBlock(prefix: String, items: Seq[ListItem]) extends Term
```

Comment:
if you renamed `item` to `items`, then why not use `terms` on line 100?

Reply:
Changed to `terms`
The reason is weird, but there are several places where parameter was named improperly. Like: `Paragraph.term`, `Text.part` ...

Reply:
yes, i think i started with `item: Type*` and don't like using plural for those.

Reply:
i have extracted this change into a separate pr, #2532.